README.md

```diff
@@ -1,22 +1,67 @@
-# Duck Game
-Duck game is rouge-like top down wave shooter.
+<h1 align="center"> Duck Game </h1>
+<!-- ![Title_Background]() -->
+<!-- TODO: Header image -->
 
-You're a duck defending your nest from hunters from killing you. 
+Duck Game is a roguelike top-down wave shooter where you, as a determined duck, must defend your nest from relentless hunters. Arm yourself with powerful weapons, unlock upgrades, and prepare to face the final boss!
 
-## Road Map
-We are a student development team and we aren't working on this game 24/7, so development will be slow but we are commited to making this game better over time. 
 
-### Current Alpha Development Map
+<!-- ## Features 
+Need a features tab. Possibly talking about the game loop and the different weapons and upgrades?
+-->
 
-27th of May: Map and Terrian Update
+## Team
+We are the Studio 5+6 Games Team! We are a team in our third year at the Bachelor of Information Technology in Otago Polytechnic. The game started development at the start of 2024 and has been in active development since then. In July, the team experienced a shuffle, with two developers moving to another project and one new developer joining.
+<!-- Note: Proper name for the team? -->
 
-17th of June: Sprite Update
+- Developers
+  - [Alex Reid](https://github.com/AlexReidNZ)
+  - [Kyle Black](https://github.com/kblack89)
+  - [Rohan Anakin](https://github.com/Loerad)
+  - [Palin Wiseman](https://github.com/PBWiseman)
+- Former developers
+  - [Lorna Hart](https://github.com/hartgit)
+  - [Jun Xu](https://github.com/YJunX)
 
-Onward: ?????
+## How to play
 
-## How To install 
-Head over to the [Releases Tab](https://github.com/BIT-Studio-4/Duck-Game/releases) and pick a version to download. 
+You can install the game from our [releases page](https://github.com/OtagoPolytechnic/Duck/releases/latest). To play, download the DuckGame.zip file, unzip it, and run DuckGame.exe.
 
-Click on the DuckGame-VX.X.X.zip file to download and then extract the files somewhere and run the StandaloneWindows.exe.
+The game is currently available only for Windows, with an arcade machine version and potential support for mobile devices in the future.
 
-We only currently have the game working for 64-bit Windows operating systems.
+| Action | Control |
+|-|-|
+| Movement | WASD | 
+| Shooting | Left click |
+| Aiming | Mouse movement | 
+
+## Get Involved
+
+Join the Snowball Slam community and stay up to date with the latest developments, bug fixes, and upcoming features:
```

Comment:
Game is Duck (working title) not snowball slam

Reply:
Dammit thought I changed all of them. Fixed.